OpenRA.Mods.Cnc/Traits/PortableChrono.cs

```diff
@@ -114,12 +114,12 @@ public void ResolveOrder(Actor self, Order order)
 					self.CancelActivity();
 
 				var cell = self.World.Map.CellContaining(order.Target.CenterPosition);
-				self.SetTargetLine(order.Target, Color.LawnGreen);
 				if (maxDistance != null)
-					self.QueueActivity(move.MoveWithinRange(order.Target, WDist.FromCells(maxDistance.Value)));
+					self.QueueActivity(move.MoveWithinRange(order.Target, WDist.FromCells(maxDistance.Value), null, Color.LawnGreen));
```

Comment:
If you specify this as `targetLineColor: Color.LawnGreen` you can skip the `null`, and make this more consistent with other activities.

Reply:
Fixed.